solidity/test/bridge/Bridge.test.ts

```diff
@@ -1423,6 +1448,704 @@ describe("Bridge", () => {
+                        () => {
+                          it("should revert", async () => {
+                            // TODO: Implementation.
+                          })
+                        }
+                      )
+                    })
+
+                    context("when there are multiple outputs", () => {
+                      context(
+                        "when output vector consists only of pending requested redemptions",
+                        () => {
+                          // TODO: Implementation.
+                        }
+                      )
+
+                      context(
+                        "when output vector consists of pending requested redemptions and a non-zero change",
+                        () => {
+                          const data: RedemptionTestData =
+                            MultiplePendingRequestedRedemptionsWithChange
+
+                          let tx: ContractTransaction
+                          let bridgeBalance: RedemptionBalanceChange
+                          let walletPendingRedemptionsValue: RedemptionBalanceChange
+                          let treasuryBalance: RedemptionBalanceChange
+                          let redeemersBalances: RedemptionBalanceChange[]
+
+                          before(async () => {
+                            await createSnapshot()
+
+                            // eslint-disable-next-line @typescript-eslint/no-extra-semi
+                            ;({
+                              tx,
+                              bridgeBalance,
+                              walletPendingRedemptionsValue,
+                              treasuryBalance,
+                              redeemersBalances,
+                            } = await runRedemptionScenario(data))
+                          })
+
+                          after(async () => {
+                            await restoreSnapshot()
+                          })
+
+                          it("should close processed redemption requests", async () => {
+                            for (
+                              let i = 0;
+                              i < data.redemptionRequests.length;
+                              i++
+                            ) {
+                              const redemptionRequest =
+                                // eslint-disable-next-line no-await-in-loop
+                                await bridge.pendingRedemptions(
+                                  buildRedemptionKey(
+                                    data.wallet.pubKeyHash,
+                                    data.redemptionRequests[i]
+                                      .redeemerOutputScript
+                                  )
+                                )
+
+                              expect(redemptionRequest.requestedAt).to.be.equal(
+                                0,
+                                `Redemption request with index ${i} has been not closed`
```

Comment:
has been not closed -> has not been closed
?

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/bd7b48fc10fcb38ee744a3df0676a2919130ab1c